### PR TITLE
校对 5.4.Xorg 配置

### PR DESCRIPTION
**Pull request by ykla**

### CODE REVIEW

**Review by rowheel — CHANGES_REQUESTED** [review]
Besides, we better change "私有内核接口" to "FreeBSD 专有内核接口", since the term here doesn't necessarily mean the implementation of certain API is close sourced, it more likely means that the APIs only exists in FreeBSD Kernel, and are not found in other Unix-like kernels.

File: di-5-zhang-xwindow-xi-tong/5.4.-xorg-pei-zhi.md [hunk 1]
```diff
@@ -2,26 +2,27 @@
 
 ## 5.4.1. 快速开始
 
-Xorg 支持大多数常规显卡、键盘和鼠标等设备。
+Xorg 支持大多数常规的显卡、键盘和鼠标等设备。
 
-> **提示**
+> **技巧**
```
[the remark this answers is not in the document]
Comment (ykla) [reply di-5-zhang-xwindow-xi-tong/5.4.-xorg-pei-zhi.md:7]: FreeBSD 专有内核接口 不仅限于  BSD。Linux windows下也都是私有的。

[the remark this answers is not in the document]
Comment (ykla) [reply di-5-zhang-xwindow-xi-tong/5.4.-xorg-pei-zhi.md:7]: 根据翻译指南，Tip  统一 为 “技巧”

File: di-5-zhang-xwindow-xi-tong/5.4.-xorg-pei-zhi.md [hunk 2]
```diff
@@ -31,118 +32,118 @@ Xorg 支持大多数常规显卡、键盘和鼠标等设备。
...
-通常不需要手动配置。请不要手动创建配置文件，除非自动配置不起作用。
+通常不需要进行手动配置。除非自动配置不起作用，否则请不要手动创建配置文件。
 
 ### 5.4.4.1. 目录
 
-Xorg 在几个目录中查找配置文件。`/usr/local/etc/X11/` 是这些文件在 FreeBSD 上的推荐目录。使用此目录有助于将应用程序文件与操作系统文件分开。
+Xorg 在多个目录中查找配置文件。**/usr/local/etc/X11/** 是这些文件在 FreeBSD 上的推荐目录。使用此目录有助于将应用程序文件与操作系统文件分开。
 
-将配置文件存储在旧的 `/etc/X11/` 中仍然有效。但是，这会将应用程序文件与基本 FreeBSD 文件混合在一起，因此不建议这样做。
+将配置文件存储在旧的 **/etc/X11/** 中仍然有效。但是，这会将应用程序文件与 FreeBSD 基本系统的文件混合在一起，因此不建议这样做。
```
[the remark this answers is not in the document]
Comment (ykla) [reply di-5-zhang-xwindow-xi-tong/5.4.-xorg-pei-zhi.md]: 改为“以往的”

File: di-5-zhang-xwindow-xi-tong/5.4.-xorg-pei-zhi.md [hunk 3]
```diff
@@ -31,118 +32,118 @@ Xorg 支持大多数常规显卡、键盘和鼠标等设备。
...
 
 ### 5.4.4.2. 单个或多个文件
 
-使用多个文件，每个文件配置一个特定的设置，比传统的单个 `xorg.conf` 更容易。这些文件存储在主配置文件目录的 `xorg.conf.d/` 子目录中。完整路径通常是 `/usr/local/etc/X11/xorg.conf.d/`。
+使用多个文件，每个文件配置一个特定的设置，比使用传统的单个 **xorg.conf** 更简单。这些文件存储在主配置文件目录的 **xorg.conf.d/** 子目录中。完整路径通常是 **/usr/local/etc/X11/xorg.conf.d/**。
```
[the remark this answers is not in the document]
Comment (ykla) [reply di-5-zhang-xwindow-xi-tong/5.4.-xorg-pei-zhi.md:63]: 是简单 的意思。不是简洁。单独的配置文件会配置起来会很复杂

File: di-5-zhang-xwindow-xi-tong/5.4.-xorg-pei-zhi.md [hunk 4]
```diff
@@ -31,118 +32,118 @@ Xorg 支持大多数常规显卡、键盘和鼠标等设备。
...
+ports 源代码库里的 **x11** 类别中提供几个 NVIDIA 驱动程序。安装与显卡相符的驱动程序。
 
 如需参考，请参阅 [https://en.wikipedia.org/wiki/List\_of\_Nvidia\_graphics\_processing\_units](https://en.wikipedia.org/wiki/List\_of\_Nvidia\_graphics\_processing\_units)。
 
-在port **x11/nvidia-driver** 或 **x11/nvidia-driver-xxx** 中可以找到对 NVIDIA 卡的内核支持。新卡使用前者，旧卡使用port -xxx，其中 xxx 是 304、340 或 390 之一，表示驱动程序的版本。对于这些具体的 -xxx，请参考支持的 [NVIDIA GPU 产品](http://download.nvidia.com/XFree86/FreeBSD-x86\_64/465.19.01/README/) 页面对应选取。此页面列出了不同版本的驱动程序支持的设备。旧版驱动程序可在 i386 和 amd64 上运行。当前驱动只支持 amd64。有关详细信息，请阅读 [NVIDIA 驱动程序的安装和配置](http://download.nvidia.com/XFree86/FreeBSD-x86\_64/465.19.01/README/)。虽然我们建议在每次内核重编译时都编译此驱动程序以获得最大的可靠性，但因为它几乎不使用私有内核接口，因此通常在内核更新时是可靠的。
+在 port **x11/nvidia-driver** 或 **x11/nvidia-driver-xxx** 中可以找到对 NVIDIA 卡的内核支持。新卡使用前者，旧卡使用 port `-xxx`，其中 xxx 是 304、340 或 390 之一，表示驱动程序的版本。对于这些具体的 `-xxx`，请参考支持的 [NVIDIA GPU 产品](http://download.nvidia.com/XFree86/FreeBSD-x86\_64/465.19.01/README/) 页面对应选取。此页面列出了不同版本的驱动程序支持的设备。旧版驱动程序可在 i386 和 amd64 上运行。当前驱动只支持 amd64。有关详细信息，请阅读 [NVIDIA 驱动程序的安装和配置](http://download.nvidia.com/XFree86/FreeBSD-x86\_64/465.19.01/README/)。虽然我们建议在每次内核重编译时都重新编译此驱动程序以获得最大的可靠性，但因为它几乎不使用私有内核接口，因此通常在内核更新时是不受影响的。
```
[the remark this answers is not in the document]
Comment (ykla) [reply di-5-zhang-xwindow-xi-tong/5.4.-xorg-pei-zhi.md]: 旧卡使用带 `-xxx` 的 port  OK

**Pull request merged**